scripts/local/.env

```diff
@@ -0,0 +1,52 @@
+# -----------------------------------------------------------------------------
+# Required Environment Variables for Local setup
+# -----------------------------------------------------------------------------
+#
+# Please locate and paste your string values here to set up 
+# 
+# DO NOT add quotation marks to values
+# -----------------------------------------------------------------------------
+
+# -----------------------------------------------------------------------------
+# IoTHub connection string:
+# {Your IoT Hub} > Shared access policies > Connection string -- primary key
+# -----------------------------------------------------------------------------
```

Comment:
it would be useful to have example with fake keys (making it clear they're fake, eg. using "1234567" or "abcdef", so nobody thinks we checked in valid credentials)

Reply:
done